selfdrive/ui/qt/widgets/keyboard.cc

```diff
@@ -126,7 +126,7 @@ Keyboard::Keyboard(QWidget *parent) : QFrame(parent) {
   std::vector<QVector<QString>> specials = {
     {"[","]","{","}","#","%","^","*","+","="},
     {"_","\\","|","~","<",">","€","£","¥","•"},
-    {"123",".",",","?","!","`",BACKSPACE_KEY},
+    {"123","'",",","?","!","`",BACKSPACE_KEY},
```

Comment:
this is supposed to be the same as the iphone keyboard, but it seems the apostrophe was mistaken for a backtick
```suggestion
    {"123",".",",","?","!","'",BACKSPACE_KEY},
```

Reply:
Thanks! Applied.
Apologies for the extra spam, I've grown used to web pages that don't dynamically reload.